lua/astrocommunity/editing-support/dial-nvim/init.lua

```diff
@@ -1,5 +1,27 @@
 return {
   "monaqa/dial.nvim",
```

Comment:
@Axlefublr Does adding lazy = true break anything ?

Reply:
At least doesn't for me. Even with `lazy = true`, the mappings work the first time I press them, as expected.

Reply:
```suggestion
  "monaqa/dial.nvim",
  lazy = true,
```

Reply:
Feel free to add it then :)

Reply:
sure! but doesn't `keys` already make it lazy? are we adding `lazy = true` as a sanity check?

Reply:
I am stupid.